MAUI/PDF-Viewer/Show-Hide.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: Show and Hide annotations in .NET MAUI PDF Viewer | Syncfusion
+description: Learn here about showing and hiding annotations in a document in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control and more.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+keywords: .net maui pdf viewer, .net maui view pdf, pdf viewer in .net maui, .net maui open pdf, maui pdf viewer, maui pdf view
+---
+# Show and Hide annotations in .NET MAUI PDF Viewer (SfPdfViewer)
+ 
+You can manage the visibility of annotations using the `Hidden` property. It helps you to hide annotations containing confidential data when sharing documents externally or presenting documents, ensuring data privacy and security. Also, it makes it easier to read the documents which are cluttered with more annotations. Like other annotation properties, you can undo and redo the hiding or showing actions. Additionally, when an annotation is locked, it cannot be hidden. Annotations that are hidden will remain hidden during import, export, printing and saving
```

Comment:
At the end of the paragraph full stop is missing

Reply:
I have corrected it now sir